source/code-generation/query-converter/convert-queries.txt

```diff
@@ -0,0 +1,161 @@
+.. _rm-convert-queries:
+
+===============
+Convert Queries
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can convert SQL queries to MongoDB syntax with query converter. 
+Query converter considers the mapping rules and schema 
+transformations defined in your project when converting your SQL code.
+
+About this Task
+---------------
+
+.. include:: /includes/fact-query-converter-generic.rst
+
+Before you Begin
+----------------
+
+.. include:: /includes/fact-query-converter-disclaimer.rst
```

Comment:
This seems like it might belong in Next Steps? I think of "Before you Begin" as something that must be done before you perform the task on the page.